governance-proposal/TSC-Policy-Draft.md

```diff
@@ -0,0 +1,31 @@
+## ***DRAFT for Public Comment***
+
+# Node.js Foundation TSC Policy
+
+The Board of Directors of Node.js Foundation has set in place the following policies to govern the operation of the Technical Steering Committee (TSC). These policies relate to release processes, technical scope, business goals, etc. The TSC will, to the best of its ability, adhere to these policies. In cases where the TSC makes a judgment that the goals of Node.js Foundation are better served by making exceptions to these policies, it is expected that the TSC will communicate these exceptions and indicate their reasons to the Board at the next Board meeting.
+
+The scope of projects chosen by the TSC will focus on the subsystems, modules, scripts, tools or infrastructure to support Node.js.
+
+Node.js Foundation will support projects focused on the following:
```

Comment:
Ok sure.

There seems to be some cross-over between regular projects and "Node.js Foundation Projects" from `TSC-Project-Lifecycle.md` (because they are referred to the same) which makes things a little confusing.